assets/src/components/routePicker.tsx

```diff
@@ -37,19 +37,38 @@ const RoutePicker = ({
     garageFilterData
   )
 
-  return (
+  return featureIsEnabled("presets_workspaces") ? (
```

Comment:
**question (non-blocking):** How long are you planning to leave this flag up? There's some duplication between the two branches that could drift apart over time, but if this is planned to be released for everybody soon, then it's not really an issue. If it's gonna be up for a while, then at least the `RoutesList` could be pulled out of the flag check.

Reply:
The idea is that this will be released to all Skate users in the coming weeks, so it won't hang around for a long time.